recipes/prince/recipe.yaml

```diff
@@ -0,0 +1,52 @@
+context:
+  name: prince
+  version: "0.15.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+
+  url: https://files.pythonhosted.org/packages/24/2d/445c241764dc5602c955d6a9b00f00953ade0ca1ce8db143d053b818e932/${{ name }}-${{ version }}.tar.gz
+  sha256: c4508347fa5433c50e8e5ef9bed63a25d8e568f4bfea16457097b7a64404bc19
+
+build:
+  noarch: python
+  script: python -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
```

Comment:
If this package is `noarch: python` (which I think it is), it should not have a `build` section with compilers.